love/src/Config.js

```diff
@@ -436,6 +436,92 @@ export const severityEnum = {
   critical: 4,
 };
 
+// MT Camera Hexapod
+export const hexapodStatusStatetoStyle = {
+  UNKNOWN: 'undefined',
+  ENABLED: 'enabled',
+  DISABLED: 'disabled',
+  FAULT: 'fault',
+  OFFLINE: 'offline',
+  STANDBY: 'standby',
+};
+
+export const hexapodCommandableByDDSStateMap = {
+  false: 'NOT COMMANDABLE',
+  true: 'COMMANDABLE',
+};
+
+export const hexapodCommandableByDDSStatetoStyle = {
+  'NOT COMMANDABLE': 'alert',
+  COMMANDABLE: 'ok',
+};
+
+export const hexapodCompensationModeStateMap = {
+  false: 'DISABLED',
+  true: 'ENABLED',
+};
+
+export const hexapodCompensationModeStatetoStyle = {
+  DISABLED: 'undefined',
+  ENABLED: 'ok',
+};
+
+export const hexapodInterlockStateMap = {
+  Disengaged: 'Disengaged',
+  Engaged: 'Engaged',
+};
+
+export const hexapodControllerStateMap = {
+  0: 'Standby',
+  1: 'Disabled',
+  2: 'Enabled',
+  3: 'Offline',
+  4: 'Fault',
+};
+
+export const hexapodControllerStatetoStyle = {
+  Standby: 'undefined',
+  Disabled: 'undefined',
+  Enabled: 'ok',
+  Offline: 'undefined',
+  Fault: 'warning',
```

Comment:
Similar as explained above, key values of this `stateToStyle` mapping should be uppercase. Line 483 should go as: `STANDBY: 'undefined`